source/extensions/common/wasm/null/null_plugin.h

```diff
@@ -71,6 +72,11 @@ class NullPlugin : public NullVmPlugin {
 
   void onCreate(uint64_t context_id, uint64_t root_context_id);
 
+  uint64_t onNewConnection(uint64_t context_id);
+  uint64_t onDownstreamData(uint64_t context_id, uint64_t data_length, uint64_t end_of_stream);
```

Comment:
'end_stream' is imperative whereas end_of_stream is indicative.  Since this is a boolean describing the situation, it is indicative, so end_stream is incorrect in this situation.  Envoy seems to use 'end_stream' in both imperative (sendLocalReply) and indicative.  Let's leave this for the upstream reviewers and perhaps we can fix envoy to use the proper mood in the proper situations.